src/Mapbender/CoreBundle/Component/Cache/Backend/File.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+
+namespace Mapbender\CoreBundle\Component\Cache\Backend;
+use Mapbender\BaseKernel;
+use Mapbender\CoreBundle\Component\Exception\CacheMiss;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Cache backend using the file system.
+ * This uses the default Symfony cache directories also used by twig, doctrine etc as the base storage location.
+ *
+ * Cache backend provides raw key:string value storage and handles automatic invalidation based on "marks"
+ * (which can be timestamps, but can also be more complex, as long as they are serializable).
+ *
+ * Instance registered in container at mapbender.presenter.application.cache.backend (see serviccs.xml)
+ */
+class File
+{
+    /** @var string */
+    protected $rootPath;
+
+    public function __construct(ContainerInterface $container)
+    {
+        /** @var BaseKernel $kernel */
+        $kernel = $container->get('kernel');
+        $rootPath = realpath($kernel->getCacheDir());
```

Comment:
It's also possible to inject the kernel.cache_dir path directly from the service configuration (services.xml) as an argument:
```
<service id="mapbender.presenter.application.cache.backend" class="%mapbender.presenter.application.cache.backend.class%">
<argument>%kernel.cache_dir%</argument> 
</service>
```
and get rid of $container and $kernel variables. It could be more readable this way, what do you think?

Reply:
Sounds good, I'll do it.

Reply:
[Done](https://github.com/mapbender/mapbender/pull/812/commits/5a0865e376ed52ed0d57a8b7434eba3bbad8b710)